router.js

```diff
@@ -0,0 +1,29 @@
+
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+
+// require our controllers
+const EntryController = require('./controllers/EntryController')
+
+// require our models that node-resful will transform
+var entryModel = require('./models/Entry')
+
+// Routes - Define routes here
+router.get('/entries', EntryController.get)
+
+// you can also for route that need a little amount of code,
+// declare the middleware directly here
```

Comment:
Two possible cases : 

- Simple HTTP handling middleware, when you want for example to render a page or send a json document that is fixed (by default, my api's have a simple json document describing the API at the root, ex: http://api.mineweb.org/).
In this case, i will add it directly in the `router` file, but i need to take less than 5 lines and never be re-used.


- The middleware need to be re-used across a lot of routes like a `isAuthenticated` or `isAdmin` middleware, in this case i will put in a file exporting all middewares, then requiring them when you need them in the `router` file.